INDEX

```diff
@@ -245,15 +246,15 @@
 		"icon": "https://icons.freenas.org/community-icons/madsonic.png",
 		"name": "MadSonic",
 		"official": false,
-		"primary_pkg": null
+		"primary_pkg": ""
```

Comment:
Why are we changing this to empty string ?

Reply:
If I remember my thinking here correctly during this PR creation I was just looking at the `index.schema` (without really much changes to the properties) and solving the issue after that. Having the `primary_pkg` property set to `null` here will give the `jsonchema` exception (note that before this change the current INDEX check didn't capture these issues):

```
None: None is not of type 'string'
```

Another way to solve this could be to simply allow null values for this property and change it to `["string", "null"]` if you would prefer that?

Reply:
Yes @fulder let's please do that as we would be changing behavior otherwise ( how it's working right now )

Reply:
Sure thing, I've changed the schema to accept `null` values for `primary_pkg`, synced the branch from master and solved some new INDEX/plugin issues :) 

Looking through the `INDEX` file again I realize there are entries with both `null` values and empty strings, but you're right, better to keep the same behavior instead of introducing possible bugs to the index.